featurebyte/models/tile_registry.py

```diff
@@ -0,0 +1,83 @@
+"""
+TileModel document model
+"""
+from __future__ import annotations
+
+from typing import List, Optional
+
+from datetime import datetime
+
+import pymongo
+from pydantic import StrictStr
+
+from featurebyte.models.base import (
+    FeatureByteCatalogBaseDocumentModel,
+    PydanticObjectId,
+    UniqueValuesConstraint,
+)
+from featurebyte.schema.common.base import BaseDocumentServiceUpdateSchema
+
+
+class TileModel(FeatureByteCatalogBaseDocumentModel):
+    """
+    TileModel document
+    """
+
+    feature_store_id: PydanticObjectId
+    tile_id: StrictStr
+    aggregation_id: StrictStr
+
+    tile_sql: StrictStr
+    entity_column_names: List[StrictStr]
+    value_column_names: List[StrictStr]
+    value_column_types: List[StrictStr]
+
+    frequency_minute: int
+    time_modulo_frequency_second: int
+    blind_spot_second: int
+
+    last_tile_start_date_online: Optional[datetime]
```

Comment:
would it make sense to extract these 4 properties to a struct as it looks like it's reused with the `TileUpdate` schema too?

Reply:
Yes, I think that makes sense.